six/modules/c++/six.sidd/include/six/sidd/SFA.h

```diff
@@ -205,8 +201,10 @@ struct SFAPolyhedralSurface : public SFASurface
         SFASurface(TYPE_NAME)
     {
     }
-    virtual ~SFAPolyhedralSurface()
+
+    virtual SFAPolyhedralSurface* clone() const
     {
+        return new SFAPolyhedralSurface(*this);
     }
 
     std::vector<mem::ScopedCopyablePtr<SFAPolygon> > patches;
```

Comment:
Actually looks like there are several other spots... you'll want to look at each one to see if it's going to slice or not.